src/Api/Response/Filter/BaseFilter.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Findologic\Api\Response\Filter;
+
+abstract class BaseFilter
+{
+    public const RATING_FILTER_NAME = 'rating';
+    public const CAT_FILTER_NAME = 'cat';
+    public const VENDOR_FILTER_NAME = 'vendor';
+
+    public ?string $displayType;
+
+    public bool $hidden = false;
+
+    public string $id;
+    public string $name;
+    /** @var FilterValue[] */
+    public array $values = [];
+    public bool $isMain;
+    public ?string $selectMode;
+    public ?string $cssClass;
+    public ?string $noAvailableFiltersText;
+    public ?string $combinationOperation;
+    public ?string $findologicFilterType;
```

Comment:
Please keep a consistent spacing across the files and especially within one file

```suggestion
    public ?string $displayType;

    public bool $hidden = false;

    public string $id;

    public string $name;

    /** @var FilterValue[] */
    public array $values = [];

    public bool $isMain;

    public ?string $selectMode;

    public ?string $cssClass;

    public ?string $noAvailableFiltersText;

    public ?string $combinationOperation;

    public ?string $findologicFilterType;
```